### PR TITLE
latex: add most of Unicode 9.0 block Mathematical Alphanumeric Symbols

### PR DESCRIPTION
**Pull request by pzorin**

### CODE REVIEW

File: tables/latex.txt [hunk 1]
```diff
@@ -198,8 +198,8 @@ _x	ₓ	0
 ^(	⁽	0
 ^)	⁾	0
 ^n	ⁿ	0
-###
-### Unicode symbols interval 2100-214F
+###
```
Comment (McSinyx) [inline tables/latex.txt]: I believe there is an unintentional trailing space here.

Comment (mike-fabian) [reply tables/latex.txt]: OK, I’ll fix the trailing space.

**Comment by mike-fabian** [comment]
Included in 1.3.10: 
https://github.com/moebiuscurve/ibus-table-others/releases/tag/1.3.10

**Comment by McSinyx** [comment]
Could you please review other pending PRs (including mine)?

**Comment by mike-fabian** [comment]
Yes, I’ll review your pull request as well.
Thank you very much for your pull request!

**Pull request merged**